modules/serverpod_auth/serverpod_auth_server/test/password_hash/argon2id_password_hash_test.dart

```diff
@@ -156,7 +159,8 @@ void main() {
         legacySalt: salt,
       );
 
-      expect(await passwordHash.validate('chaser1'), isFalse);
+      expect(await passwordHash.validate('chaser1'),
+          isNot(isA<PasswordValidationSuccess>()));
```

Comment:
Chore: Update this for all tests that are expected to fail validation.

Remember to update the descriptions, we are not returning `true` or `false` anymore.

```suggestion
          isA<PasswordValidationFailed>());
```